data/hai/hai reveal 3 early.txt

```diff
@@ -0,0 +1,462 @@
+				decline
+			label ordeal
+			`You land on the world of <planet> and do your part to sell the opportunity on Mountaintop. You struggle to maintain a cheerful and enthusiastic facade as you count down the minutes until this ordeal is over.`
+				decline
+			label interminable
+			`You land on the world of <planet> and do your part to sell the opportunity on Mountaintop. Why did you ever agree to this interminable dog and pony show? Is it really worth all this just to save one tiny little galaxy?`
+				decline
+			label toomany
+			`You land on the world of <planet> and do your part to sell the opportunity on Mountaintop. You're pretty sure you've done this more times than there are habitable planets in Hai space. Did you lose track and repeat a world? Will this nightmare ever end?`
+				decline
+			label overdone
+			`You land on the world of <planet> and do your part to sell the opportunity on Mountaintop. Terry makes good on her promise of forcing you to shuffle out and do some last-minute spruiking to offload the remaining promotional materials to those who missed it the first time.`
+				decline
+
+
+
+mission "Hai Building Mountaintop [3] Furnishings"
+	name "Retrieve human furnishings."
+	description "The volunteers are so numerous that the Hai need outside help transporting furnishings for their quarters to the new station."
+	autosave
+	landing
+	source "Mountaintop"
+	stopover "Prime"
+	destination "Mountaintop"
+	clearance
+	on visit
+		dialog `You have arrived at the Mountaintop construction site, but it isn't finished or staffed yet.`
+	to offer
+		has "Hai Building Mountaintop [2] Recruits: done"
+	cargo "human furnishings" 200
+	blocked `Sayari contacts you and says, "<first>, you're going to need 200 tons of cargo space for this." You should return here after freeing up <capacity>.`
+	on offer
+		conversation
+			`As you and Terry offload her remaining supplies on Mountaintop where she intends to manage the arrivals and setup personally, you get a communication from Sayari come to your personal device.`
+			`	"Captain <last>! Glad I could get a hold of you on the other side of the wormhole there. I see the new relays are working. We've just discovered a problem: interest has soared in sign-ups for Mountaintop, but we've realized that most of our 'human' furnishings produced in Hai space aren't going to pass as having been manufactured with your technology. Mr Yang has pulled some strings, but now we urgently need someone to go pick them up from Prime in Betelgeuse. Everything else should be ready on our end soon enough, so if you can just handle that pick-up and delivery we'll be okay."`
+			`	You let her know you'll handle it and prepare to leave.`
+				accept
+	on stopover
+		dialog `You have the furnishings the Hai ordered. It's time to return them to <destination>.`
+	on complete
+		event "role-play station active" 12
+		event "news: hai video clip debunked"
```

Comment:
Really? Where? How?

Reply:
If this is referring to the event, it's a background event that feeds into News and other things.

Reply:
Seems like the debunking should be something made explicit in the story (although with the movie production idea the debunking would be mentioned as saying it was marketing material).